notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1224 @@
+{
```

Comment:
<p>You can add a sentence like this:</p><p> </p><p>Learn about <a href="https://docs.haystack.deepset.ai/docs/custom-components" target="_blank">creating custom components</a> in our documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
done